.github/workflows/test_objectivec.yml

```diff
@@ -27,24 +36,32 @@ jobs:
         - platform: "iOS"
           destination: "platform=iOS Simulator,name=iPhone 13,OS=latest"
           xc_project: "ProtocolBuffers_iOS.xcodeproj"
+        # We run presubmits on all "Debug" entries, but not on "Release" entries
+        - xc_config: "Debug"
+          presubmit: true
+        - xc_config: "Release"
+          presubmit: false
 
-    name: Xcode ${{ matrix.platform}} ${{ matrix.xc_config }}
+    name: ${{ !matrix.presubmit && inputs.continuous-prefix || '' }} Xcode ${{ matrix.platform}} ${{ matrix.xc_config }}
     runs-on: macos-12
     env:
       DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
     steps:
       - name: Checkout pending changes
+        if: ${{ !matrix.presubmit || inputs.test-type == 'continuous' }}
         uses: protocolbuffers/protobuf-ci/checkout@v3
         with:
           ref: ${{ inputs.safe-checkout }}
 
       - name: Setup ccache
+        if: ${{ matrix.presubmit || inputs.test-type == 'continuous' }}
```

Comment:
My previous comment about parsing and validating these yaml files got me thinking.  We could probably enforce that every step has this line in it :). Similar thing with the naming scheme

Reply:
Discussed offline, yml parsing will be a follow up project.